### ISSUE TITLE
Small typo errors including format of WCAG contrast 4.5:1

### ISSUE THREAD

**Issue by MI6-94** [mi6/ic-design-system#1031]
## Summary of the bug

There is a small typo in contrast recommendations. Says 4:5:1, it should be 4.5:1.

There is also a typo on the switch documentation page. It should be switch's' not 'switches'
